Automattic/wp-calypso
# Hosting Metrics: Add "Learn more" link to header on site monitoring page

Closes https://github.com/Automattic/dotcom-forge/issues/3492

## Proposed Changes

This PR adds "Learn More" link to the header on Site Monitoring page:

<img width="638" alt="Screenshot 2023-08-23 at 9 31 02 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/f6d19bcf-214c-4078-93e3-ff038fcc1d9d">

## Testing Instructions

* Make sure you have an Atomic site
* Navigate to `Tools > Site Monitoring` in your dashboard 
* Check that "Learn more" link is present in the header after the text 'Real time information to troubleshoot or debug problems with your site'

The issue description [here](https://github.com/Automattic/dotcom-forge/issues/3492) does not have `.` at the end after `Learn more` but I checked similar headers that we have and it seems to have it on other headers such as `https://wordpress.com/settings/performance/{yourAtomicSite}` so I kept it in there. Happy to remove it if that's the preference. 

## Pre-merge Checklist
